server/src/test/java/io/crate/integrationtests/CreateTableIntegrationTest.java

```diff
@@ -194,22 +195,21 @@ tn boolean as (ta = false)
                     from information_schema.columns
                     where table_name = 't'
                     order by 2""");
-        assertThat(printedTable(response.rows())).isEqualTo(
-             """
-             ta| 1
-             tb| 2
-             tc| 3
-             tc['td']| 4
-             tc['te']| 5
-             tc['tf']| 6
-             tc['tf']['tg']| 7
-             ti| 9
-             ti['tj']| 10
-             ti['tk']| 11
-             ti['tk']['tl']| 12
-             tm| 13
-             tn| 14
-             """); // 'th' is a named index and is assigned column position 8
+        assertThat(response).hasRows(
+             "ta| 1",
+             "tb| 2",
+             "tc| 3",
+             "tc['td']| 4",
+             "tc['te']| 5",
```

Comment:
The new approach changes the way nested columns are incremented.

I think that should be fine and it still fits the "in definition" order criteria.

Reply:
but for this test, there is no change, or am I not seeing it?

Reply:
Right, I commented it on the wrong one, it's visible in the others.